### PR TITLE
[res] TensorFlowPythonExamples for PadV2

### PR DESCRIPTION
**Pull request by a-zh**
This will introduce TensorFlowPythonExamples for PadV2 op

Draft: #3566
Issue: #3362

### CODE REVIEW

File: res/TensorFlowPythonExamples/examples/PadV2/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import tensorflow as tf
+import numpy as np
+
+input_ = np.ones(shape=[1, 1, 1, 1], dtype=np.float32)
+paddings_ = np.array([[1, 1], [2, 2], [3, 3], [4, 4]], dtype=np.int32)
+constant_values_ = 1
+op_ = tf.raw_ops.PadV2(input=input_, paddings=paddings_, constant_values=constant_values_)
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/PadV2/__init__.py:7]: With TF 1.13.1, I got this error
```
AttributeError: module 'tensorflow' has no attribute 'raw_ops'
```

Comment (a-zh) [reply res/TensorFlowPythonExamples/examples/PadV2/__init__.py:7]: This is expected behavior for TF less 2.0. Do you want more readable message for test?

Comment (seanshpark) [reply res/TensorFlowPythonExamples/examples/PadV2/__init__.py:7]: Our default TF is 1.13.1. If fails in 1.13.1 we cannot use this example.

Comment (a-zh) [reply res/TensorFlowPythonExamples/examples/PadV2/__init__.py:7]: I'm sorry. I cannot find `PadV2` for `TF 1.13.2`. `PadV2` - a new operator which not available on `TF 1.13.2`. I don't have any ideas for fix it.

Comment (seanshpark) [reply res/TensorFlowPythonExamples/examples/PadV2/__init__.py:7]: Please add a comment at the top like
```
# NOTICE Run this script in TensorFlow 2.x as PadV2 is not available in 1.13.2
```

Comment (binarman) [reply res/TensorFlowPythonExamples/examples/PadV2/__init__.py]: @seanshpark 
I checked this operator in tf 1.13 again and found solution, now it works.

**Comment by seanshpark** [comment]
@binarman , thanks!

**Comment by binarman** [comment]
FYI @seanshpark 

@kvochko @AShedko and @a-zh are on vacation for two weeks (will be back 24th August)
We can not get approves from them so I think we should remove/ignore their status in existing PRs

**Pull request merged**